src/test/java/org/fogbowcloud/manager/core/plugins/identity/ldap/TestLdapIdentityPlugin.java

```diff
@@ -127,8 +127,8 @@ public void testCreateTokenFail() throws Exception {
 
 			ldapStoneIdentity.createToken(userCredentials);
 			Assert.fail();
-		} catch (Exception e) {
-			Assert.assertEquals(HttpStatus.UNAUTHORIZED.toString(), e.getMessage());
+		} catch (UnauthorizedException e) {
+			Assert.assertEquals("Couldn't load account summary from LDAP Server.", e.getMessage());
```

Comment:
Please set the string constant _"Couldn't load account summary from LDAP Server."_ as a _public static final String_ attribute of the LdapIdentityPlugin class. This change will help to prevent that the test will not crash in case of any modification on these message.

Reply:
We have to discuss more about that, I don't think it's a good idea creating a constant to store this kind of message.

Reply:
If you guys do not reach an agreement fast, I suggest to postpone the discussion to the last review (the future PR into develop branch) and accept the code as @ordansantos submitted (obviously, please keep note of @paulofelipefeitosa suggestion somewhere -- maybe in the code itself)